tests/test_main.cpp

```diff
@@ -248,6 +253,12 @@ struct GodotTestCaseListener : public doctest::IReporter {
 			audio_server->init();
 			return;
 		}
+
+		if (suite_name.find("[Navigation]") != -1 && navigation_server_2d == nullptr && navigation_server_3d == nullptr) {
```

Comment:
These servers are already created up on line 231 and 232, I don't believe this initialization is actually doing anything.

Reply:
lines 231 and 232 are called for tests with `[SceneTree]` substring which is not the case in the case of server-only tests from `[Navigation]` test suite